_posts/blog/2020-02-11-accumulo-spark-connector.md

```diff
@@ -0,0 +1,192 @@
+- Column pruning based on selected fields transparently reduces the amount of data returned from Accumulo
+- Server side inference: ML model inference can run on the Accumulo nodes using MLeap to increase the scalability of AI solutions as well as keeping data in Accumulo
+
+## Use-cases
+There are many scenarios where use of this connector provides advantages, below we list a few common use-cases.
+
+**Scenario 1**: A data analyst needs to execute model inference on large amount of data in Accumulo.<br>
+**Benefit**: Instead of transferring all the data to a large Spark cluster to score using a Spark model, the model can be exported and pushed down using the connector to run on the Accumulo cluster. This can reduce the need for a large Spark cluster as well as the amount of data transferred between systems, and can improve inference speeds (>2x speedups observed).
+
+**Scenario 2**: A data scientist needs to train a Spark model on a large amount of data in Accumulo.<br>
+**Benefit**: Instead of pulling all the data into a large Spark cluster and restructuring the format to use Spark ML Lib tools, the connector allows for data to be streamed into Spark as a DataFrame reducing time to train and Spark cluster size / memory requirements.
+
+**Scenario 3**: A data analyst needs to perform ad hoc analysis on large amounts of data stored in Accumulo.<br>
+**Benefit**: Instead of pulling all the data into a large Spark cluster, the connector allows for both rows and columns to be pruned using pushdown filtering with a flexible expression language.
+
+# Architecture
+The Accumulo-Spark connector is composed of two components:
+
+- Accumulo server-side iterator performs
+  - column pruning
+  - row-based filtering
+  - [MLeap](https://github.com/combust/mleap) ML model inference and
+  - row assembly using [Apache AVRO](https://avro.apache.org/)
+- Spark DataSource V2 
+  - determines the number of Spark tasks based on available Accumulo table splits
+  - translates Spark filter conditions into a [JUEL](http://juel.sourceforge.net/) expression
+  - configures the Accumulo iterator
+  - deserializes the AVRO payload
+
+<img class="blog-img-center" src="/images/blog/202002_masc/architecture.svg">
+
+# Usage
+More detailed documentation on installation and use is available in the 
+[Connector documentation](https://github.com/microsoft/masc/blob/master/connector/README.md)
+
+## Dependencies
+- Java 8
+- Spark 2.4.3+
+- Accumulo 2.0.0+
+
+JARs available on Maven Central Repository:
+- [![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.microsoft.masc/microsoft-accumulo-spark-datasource/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.microsoft.masc/microsoft-accumulo-spark-datasource) [Spark DataSource](https://mvnrepository.com/artifact/com.microsoft.masc/microsoft-accumulo-spark-datasource)
```

Comment:
If pointing to an artifact in Maven Central, it's probably best to use Maven Central's own UI (search.maven.org) rather than the 3rd party mvnrepository.com

Reply:
done